source/playgrounds.txt

```diff
@@ -177,6 +181,26 @@ of your playground.
 immediately above your selected section. Click this link to test and
 troubleshoot specific lines or sections of your playground.
 
+Code Formatting and Linting Tools
+---------------------------------
+
+MongoDB Playgrounds are proper JavaScript files and can work with popular
```

Comment:
Do we need the word "proper" here?

Reply:
I think I was trying to drive home the point here, that VS Code will treat these new file extension exactly the same as javascript files. But perhaps it reads better without. Removed.